src/components/MultiHopTrade/components/TradeConfirm/TradeConfirm.tsx

```diff
@@ -478,7 +478,9 @@ export const TradeConfirm = () => {
                     <RawText>{`1 ${sellAsset?.symbol ?? ''} = ${firstNonZeroDecimal(
                       bnOrZero(tradeQuoteStep?.rate),
                     )} ${buyAsset?.symbol}`}</RawText>
-                    {!!swapperName && <RawText color='text.subtle'>@{swapperName}</RawText>}
+                    {!!swapperName && (
+                      <RawText color='text.subtle'>@{tradeQuoteStep.source}</RawText>
```

Comment:
Not sure if we want this vs. the SwapperName, see with Li.Fi

<img width="634" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f6a67ee3-54c2-4760-87ec-6d5b37df9234">

That is technically true currently as we only support single steps quotes, i.e the step source is the underlying swapper name, but that will not hold true as we support actual multi-hop quotes.

With that being said, given the UI will change when we implement Li.Fi quotes with multiple underlying steps, happy to keep it as-is for now

Added a smallish comment in 9b09e6cb7 to ensure we keep this in mind when multi-hop quotes are in